modules/hello-world/examples/index-hello-world.js

```diff
@@ -0,0 +1,102 @@
+const couchbase = require('couchbase')
+
+async function go() {
+  const cluster = await couchbase.connect('couchbase://localhost', {
+    username: 'Administrator',
+    password: 'password',
+  })
+
+  console.log('[primary]')
+  // tag::primary[]
+  await cluster.queryIndexes().createPrimaryIndex(
+    'travel-sample',
+    // Don't error if the primary index already exists.
+    { ignoreIfExists: true }
+  )
+  // end::primary[]
+  console.log('Index creation complete')
+
+  console.log('\n[named-primary]')
+  // tag::named-primary[]
+  await cluster.queryIndexes().createPrimaryIndex(
+    'travel-sample',
+    { name: "named_primary_index" }
+  )
+  // end::named-primary[]
+  console.log('Named primary index creation complete')
+
+  console.log('\n[secondary]')
+  // tag::secondary[]
+  await cluster.queryIndexes().createIndex(
+    'travel-sample', 
+    'index_name', 
+    ['name']
+  )
+  // end::secondary[]
+  console.log('Index creation complete')
+
+  console.log('\n[composite]')
+  // tag::composite[]
+  await cluster.queryIndexes().createIndex(
+    'travel-sample',
+    'index_travel_info', ['name','id','icao','iata']
+  )
+  // end::composite[]
+  console.log('Index creation complete')
+
+  console.log('\n[drop-primary]')
+  // tag::drop-primary[]
+  await cluster.queryIndexes().dropPrimaryIndex('travel-sample')
+  // end::drop-primary[]
+  console.log('Primary index deleted successfully')
```

Comment:
Note, this doesn't seem to work for some reason. Though looking at the [API doc](https://docs.couchbase.com/sdk-api/couchbase-node-client/classes/QueryIndexManager.html#createPrimaryIndex), I would have expected this to work without needing to provide the index name option?

This is the error I get:
```
ParsingFailureError: parsing failure
    at Object.translateCppError (/Users/mariashodunke/Couchbase/docs/docs-sdk-nodejs/node_modules/couchbase/dist/bindingutilities.js:194:20)
    at /Users/mariashodunke/Couchbase/docs/docs-sdk-nodejs/node_modules/couchbase/dist/connection.js:248:54 {
  cause: LibcouchbaseError: libcouchbase error 208
      at Object.translateCppError (/Users/mariashodunke/Couchbase/docs/docs-sdk-nodejs/node_modules/couchbase/dist/bindingutilities.js:174:21)
      at /Users/mariashodunke/Couchbase/docs/docs-sdk-nodejs/node_modules/couchbase/dist/connection.js:248:54 {
    code: 208
  },
  context: QueryErrorContext {
    first_error_code: 3000,
    first_error_message: 'syntax error - at `travel-sample`',
    statement: 'DROP PRIMARY INDEX `travel-sample`',
    client_context_id: 'd4f981737ca79986',
    parameters: '',
    http_response_code: 400,
    http_response_body: ''
  }
```

@brett19 Is this a bug or do we have to provide the index name?

Reply:
Not sure if it's relevant, but the N1QL statement given in that error message is incorrect. The correct statement would be

```
DROP PRIMARY INDEX ON `travel-sample`;
```

Reply:
Asked in #sdk-and-connectors chat, looks like this is a bug (https://issues.couchbase.com/browse/JSCBC-972).